apis/s3/src/main/java/org/jclouds/s3/blobstore/S3BlobStore.java

```diff
@@ -35,6 +35,7 @@
 import org.jclouds.blobstore.functions.BlobToHttpGetOptions;
```

Comment:
In several of these files (including this one) I had to ask Eclipse to format the source because the indentation used by the project was... all over the place.  That led to some extra noise on the PR... apologies for that.